tests/e2e/README.md

```diff
@@ -13,16 +13,70 @@ design allows for the opportunity of testing chain upgrades in the
 future by providing an older Osmosis version to the container,
 performing the chain upgrade, and running the latest test suite.
 
-The file e2e\_suite\_test.go defines the testing suite and contains the
+The file `e2e_suite_test.go` defines the testing suite and contains the
 core bootstrapping logic that creates a testing environment via Docker
 containers. A testing network is created dynamically with 2 test
 validators.
 
-The file e2e\_test.go contains the actual end-to-end integration tests
+The file `e2e_test.go` contains the actual end-to-end integration tests
 that utilize the testing suite.
 
-Currently, there is a single test in `e2e_test.go` to query the balances
-of a validator.
+Currently, there is a single IBC test in `e2e_test.go`.
+
+Additionally, there is an ability to disable certain components
+of the e2e suite. This can be done by setting the environment
+variables. See "Environment variables" section below for more details.
+
+## How It Works
+
+Conceptually, we can split the e2e setup into 2 parts:
+
+1. Chain Initialization
+
+    This part can either be local or with the upgrade.
+
```

Comment:
I'm not sure if calling this "local or with the upgrade" would make a ton of sense to someone who hasn't been involved with this project from the start. Maybe we could say "either be initialized with your local changes, or be initialized with the previous osmosis version and then upgraded to your locally compiled version". I know you eventually get to this point, but this sentence had me reading it a couple of times before I realized what we meant.